src/main/java/rs/teslaris/core/dto/user/RegistrationRequestDTO.java

```diff
@@ -31,11 +32,8 @@ public class RegistrationRequestDTO {
     @Positive(message = "Preferred languageID must be a positive number.")
     private Integer preferredLanguageId;
 
-    @NotNull(message = "You must provide an authority ID.")
-    @Positive(message = "Authority ID must be a positive number.")
-    private Integer authorityId;
-
     @NotNull(message = "You must provide a person ID.")
+    @PositiveOrZero(message = "Person ID cannot be a negative number.")
     private Integer personId;
```

Comment:
I can't remember why we defined this field in this way? Why is it a "not mandatory" field? So, we have to define personId field, but zero value means it is not defined, right?

Reply:
Done.